<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wireshark</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="goToFrame">
      <layout class="QHBoxLayout" name="goToHB">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="goToLineEdit">
         <property name="inputMask">
          <string>900000000; </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="goToGo">
         <property name="text">
          <string>Go to packet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="goToCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="mainStack">
      <widget class="MainWelcome" name="welcomePage"/>
      <widget class="QWidget" name="packetPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpenRecentCaptureFile">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="actionDummyNoFilesFound"/>
    </widget>
    <widget class="QMenu" name="menuFile_Set">
     <property name="title">
      <string>File Set</string>
     </property>
     <addaction name="actionList_Files"/>
     <addaction name="actionNext_File"/>
     <addaction name="actionPrevious_File"/>
    </widget>
    <addaction name="actionFileOpen"/>
    <addaction name="menuOpenRecentCaptureFile"/>
    <addaction name="actionFileMerge"/>
    <addaction name="actionFileImport"/>
    <addaction name="actionFileClose"/>
    <addaction name="separator"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="menuFile_Set"/>
    <addaction name="separator"/>
    <addaction name="actionFileExportPackets"/>
    <addaction name="actionFileExportPacketDissections"/>
    <addaction name="actionFileExportPacketBytes"/>
    <addaction name="actionFileExportSSLSessionKeys"/>
    <addaction name="actionFileExportObjects"/>
    <addaction name="separator"/>
    <addaction name="actionFilePrint"/>
    <addaction name="separator"/>
    <addaction name="actionFileQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <addaction name="actionCaptureInterfaces"/>
    <addaction name="actionCaptureOptions"/>
    <addaction name="actionStartCapture"/>
    <addaction name="actionStopCapture"/>
    <addaction name="actionCaptureRestart"/>
    <addaction name="actionCaptureCaptureFilters"/>
    <addaction name="actionCaptureRefreshInterfaces"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuHelpManualPages">
     <property name="title">
      <string>Manual pages</string>
     </property>
     <addaction name="actionHelpMPWireshark"/>
     <addaction name="actionHelpMPWireshark_Filter"/>
     <addaction name="separator"/>
     <addaction name="actionHelpMPTShark"/>
     <addaction name="actionHelpMPRawShark"/>
     <addaction name="actionHelpMPDumpcap"/>
     <addaction name="actionHelpMPMergecap"/>
     <addaction name="actionHelpMPEditcap"/>
     <addaction name="actionHelpMPText2cap"/>
    </widget>
    <addaction name="actionHelpContents"/>
    <addaction name="menuHelpManualPages"/>
    <addaction name="separator"/>
    <addaction name="actionHelpWebsite"/>
    <addaction name="actionHelpFAQ"/>
    <addaction name="actionHelpAsk"/>
    <addaction name="actionHelpDownloads"/>
    <addaction name="separator"/>
    <addaction name="actionHelpWiki"/>
    <addaction name="actionHelpSampleCaptures"/>
    <addaction name="separator"/>
    <addaction name="actionHelpAbout"/>
   </widget>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>Go</string>
    </property>
    <addaction name="actionGoGoToPacket"/>
    <addaction name="separator"/>
    <addaction name="actionGoNextPacket"/>
    <addaction name="actionGoPreviousPacket"/>
    <addaction name="actionGoFirstPacket"/>
    <addaction name="actionGoLastPacket"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionViewExpandSubtrees"/>
    <addaction name="actionViewExpandAll"/>
    <addaction name="actionViewCollapseAll"/>
    <addaction name="separator"/>
    <addaction name="actionViewReload"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuGo"/>
   <addaction name="menuCapture"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea|Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartCapture"/>
   <addaction name="actionStopCapture"/>
   <addaction name="separator"/>
  </widget>
  <widget class="MainStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="utilityToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionFileOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a capture file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Wireshark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCaptureInterfaces">
   <property name="text">
    <string>&amp;Interfaces...</string>
   </property>
   <property name="toolTip">
    <string>Show interface details</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionStartCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../image/toolbar.qrc">
     <normaloff>:/toolbar/main/capture_start_24.png</normaloff>
     <normalon>:/toolbar/main/capture_start_active_24.png</normalon>:/toolbar/main/capture_start_24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Start</string>
   </property>
   <property name="toolTip">
    <string>Start capturing packets</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionStopCapture">
   <property name="icon">
    <iconset resource="../../image/toolbar.qrc">
     <normaloff>:/toolbar/main/capture_stop_24.png</normaloff>:/toolbar/main/capture_stop_24.png</iconset>
   </property>
   <property name="text">
    <string>S&amp;top</string>
   </property>
   <property name="toolTip">
    <string>Stop capturing packets</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close the current capture file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionDummyNoFilesFound">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No files found</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="icon">
    <iconset>
     <normaloff>:/trolltech/styles/commonstyle/images/standardbutton-help-32.png</normaloff>:/trolltech/styles/commonstyle/images/standardbutton-help-32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelpMPWireshark">
   <property name="text">
    <string>Wireshark</string>
   </property>
  </action>
  <action name="actionHelpMPWireshark_Filter">
   <property name="text">
    <string>Wireshark Filter</string>
   </property>
  </action>
  <action name="actionHelpMPTShark">
   <property name="text">
    <string>TShark</string>
   </property>
  </action>
  <action name="actionHelpMPRawShark">
   <property name="text">
    <string>RawShark</string>
   </property>
  </action>
  <action name="actionHelpMPDumpcap">
   <property name="text">
    <string>Dumpcap</string>
   </property>
  </action>
  <action name="actionHelpMPMergecap">
   <property name="text">
    <string>Mergecap</string>
   </property>
  </action>
  <action name="actionHelpMPEditcap">
   <property name="text">
    <string>Editcap</string>
   </property>
  </action>
  <action name="actionHelpMPText2cap">
   <property name="text">
    <string>Text2cap</string>
   </property>
  </action>
  <action name="actionHelpWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="actionHelpFAQ">
   <property name="text">
    <string>FAQ's</string>
   </property>
  </action>
  <action name="actionHelpDownloads">
   <property name="text">
    <string>Downloads</string>
   </property>
  </action>
  <action name="actionHelpWiki">
   <property name="icon">
    <iconset resource="../../image/toolbar.qrc">
     <normaloff>:/menu/help/wiki_24.xpm</normaloff>:/menu/help/wiki_24.xpm</iconset>
   </property>
   <property name="text">
    <string>Wiki</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelpSampleCaptures">
   <property name="text">
    <string>Sample Captures</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="../../image/toolbar.qrc">
     <normaloff>:/menu/help/wsicon16.xpm</normaloff>:/menu/help/wsicon16.xpm</iconset>
   </property>
   <property name="text">
    <string>&amp;About Wireshark</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelpAsk">
   <property name="icon">
    <iconset resource="../../image/toolbar.qrc">
     <normaloff>:/menu/help/wsicon-ask.png</normaloff>:/menu/help/wsicon-ask.png</iconset>
   </property>
   <property name="text">
    <string>Ask (Q&amp;&amp;A)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGoNextPacket">
   <property name="text">
    <string>Next Packet</string>
   </property>
   <property name="toolTip">
    <string>Go to the next packet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Down</string>
   </property>
  </action>
  <action name="actionGoPreviousPacket">
   <property name="text">
    <string>Previous Packet</string>
   </property>
   <property name="toolTip">
    <string>Go to the previous packet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Up</string>
   </property>
  </action>
  <action name="actionGoFirstPacket">
   <property name="text">
    <string>First Packet</string>
   </property>
   <property name="toolTip">
    <string>Go to the first packet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Home</string>
   </property>
  </action>
  <action name="actionGoLastPacket">
   <property name="text">
    <string>Last Packet</string>
   </property>
   <property name="toolTip">
    <string>Go to the last packet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+End</string>
   </property>
  </action>
  <action name="actionViewExpandSubtrees">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>E&amp;xpand Subtrees</string>
   </property>
   <property name="toolTip">
    <string>Expand the current packet detail</string>
   </property>
   <property name="shortcut">
    <string>Shift+Right</string>
   </property>
  </action>
  <action name="actionViewExpandAll">
   <property name="text">
    <string>&amp;Expand All</string>
   </property>
   <property name="toolTip">
    <string>Expand packet details</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionViewCollapseAll">
   <property name="text">
    <string>Collapse &amp;All</string>
   </property>
   <property name="toolTip">
    <string>Collapse all packet details</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionGoGoToPacket">
   <property name="text">
    <string>Go to Packet...</string>
   </property>
   <property name="toolTip">
    <string>Go to specified packet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionFileMerge">
   <property name="text">
    <string>&amp;Merge...</string>
   </property>
   <property name="toolTip">
    <string>Merge one or more files</string>
   </property>
  </action>
  <action name="actionFileImport">
   <property name="text">
    <string>&amp;Import...</string>
   </property>
   <property name="toolTip">
    <string>Import a file</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="toolTip">
    <string>Save as a different file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionFileExportPackets">
   <property name="text">
    <string>Export Packets...</string>
   </property>
   <property name="toolTip">
    <string>Export specified packets</string>
   </property>
  </action>
  <action name="actionFileExportPacketDissections">
   <property name="text">
    <string>Export Packet Dissections...</string>
   </property>
  </action>
  <action name="actionFileExportPacketBytes">
   <property name="text">
    <string>Export Packet &amp;Bytes...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionFileExportSSLSessionKeys">
   <property name="text">
    <string>Export SSL Session Keys...</string>
   </property>
  </action>
  <action name="actionFileExportObjects">
   <property name="text">
    <string>Export Objects...</string>
   </property>
  </action>
  <action name="actionFilePrint">
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionList_Files">
   <property name="text">
    <string>List Files</string>
   </property>
  </action>
  <action name="actionNext_File">
   <property name="text">
    <string>Next File</string>
   </property>
  </action>
  <action name="actionPrevious_File">
   <property name="text">
    <string>Previous File</string>
   </property>
  </action>
  <action name="actionViewReload">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload the current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCaptureOptions">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
   <property name="toolTip">
    <string>Capture options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionCaptureCaptureFilters">
   <property name="text">
    <string>Capture &amp;Filters...</string>
   </property>
   <property name="toolTip">
    <string>Capture filters</string>
   </property>
  </action>
  <action name="actionCaptureRefreshInterfaces">
   <property name="text">
    <string>Refresh Interfaces</string>
   </property>
   <property name="toolTip">
    <string>Refresh interfaces</string>
   </property>
  </action>
  <action name="actionCaptureRestart">
   <property name="text">
    <string>&amp;Restart</string>
   </property>
   <property name="toolTip">
    <string>Restart current capture</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainStatusBar</class>
   <extends>QStatusBar</extends>
   <header>main_status_bar.h</header>
  </customwidget>
  <customwidget>
   <class>MainWelcome</class>
   <extends>QFrame</extends>
   <header>main_welcome.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../image/toolbar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFileQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
